DashKit/Mocks/MockPodStatus.swift

```diff
@@ -380,6 +387,10 @@ extension MockPodStatus: RawRepresentable {
         if let alarmReferenceCode = alarmReferenceCode {
             rawValue["alarmReferenceCode"] = alarmReferenceCode
         }
+        
+        if let rawPodCommState = try? JSONEncoder().encode(podCommState) {
```

Comment:
[nit] I believe this can be shortened to just `rawValue["podCommState"] = try? JSONEncoder().encode(podCommState)`